packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/style.scss

```diff
@@ -46,3 +46,8 @@
 .edit-site-sidebar-navigation-screen-navigation-menus__helper-block-editor {
 	display: none;
 }
+
+.edit-site-sidebar-navigation-screen-navigation-menus {
+	margin-left: -$grid-unit-20;
+	margin-right: -$grid-unit-20;
+}
```

Comment:
Let's move this CSS class [here](https://github.com/WordPress/gutenberg/pull/67384/files#diff-ed0c8c20650bc9a6acdd0da8c3e459af70b86b90a52a9550f628423b7b63b5d7R4) as well.